source/_docs/automation/condition.markdown

```diff
@@ -4,7 +4,7 @@ description: "Automations can test conditions when invoked."
 redirect_from: /getting-started/automation-condition/
 ---
 
-Conditions are an optional part of an automation rule and can be used to prevent an action from happening when triggered. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.
+Conditions are an optional part of an automation rule and can be used to prevent an action from happening when triggered. When a condition does not return true, the script will stop executing. Conditions look very similar to triggers but are very different. A trigger will look at events happening in the system while a condition only looks at how the system looks right now. A trigger can observe that a switch is being turned on. A condition can only see if a switch is currently on or off.
```

Comment:
Let's try avoiding the term "script" here, it might confuse users. Maybe just "automation"?

Reply:
Done.